src/Sarif.UnitTests/Core/StackTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    [TestClass]
+    public class StackTests
+    {
+        [TestMethod]
+        public void Stack_CreateFromStackTrace()
+        {
+            var dotNetStack = new StackTrace();
+            Stack stack = Stack.Create(dotNetStack);
+
+            // The .NET StackTrace.ToString() override must preserve a trailing NewLine
+            // for compatibility reasons. We do not retain this behavior in ToString()
+            // but provide a facility for adding the trailing NewLine
+            Assert.AreEqual(dotNetStack.ToString(), stack.ToString(StackFormat.TrailingNewLine));
```

Comment:
I won't push on this if you guys both like FluentAssertions better, though.

---
In reply to: [60634645](https://github.com/Microsoft/sarif-sdk/pull/205#discussion_r60634645) [](ancestors = 60634645,60628907)